src/node.cc

```diff
@@ -1380,7 +1381,19 @@ Isolate* NewIsolate(ArrayBufferAllocator* allocator, uv_loop_t* event_loop) {
   isolate->SetFatalErrorHandler(OnFatalError);
   isolate->SetAllowWasmCodeGenerationCallback(AllowWasmCodeGenerationCallback);
   v8::CpuProfiler::UseDetailedSourcePositionsForProfiling(isolate);
+  isolate->SetPromiseRejectCallback(task_queue::PromiseRejectCallback);
 
+  // NOTE: the following two callbacks currently CHECKs that the isolates
+  // have associated environments.
+  isolate->SetHostInitializeImportMetaObjectCallback(
+      loader::ModuleWrap::HostInitializeImportMetaObjectCallback);
+  isolate->SetHostImportModuleDynamicallyCallback(
+      loader::ModuleWrap::ImportModuleDynamically);
+
+#if defined HAVE_DTRACE || defined HAVE_ETW
```

Comment:
Since you're here: `#if defined(HAVE_DTRACE) || defined(HAVE_ETW)` - the paren-less style is unidiomatic.